manim/mobject/opengl/opengl_vectorized_mobject.py

```diff
@@ -1320,22 +1314,31 @@ def pointwise_become_partial(
         i2 = nppc * (lower_index + 1)
         i3 = nppc * upper_index
         i4 = nppc * (upper_index + 1)
-
         vm_points = vmobject.points
         new_points = vm_points.copy()
         if num_curves == 0:
             new_points[:] = 0
             return self
         if lower_index == upper_index:
             tup = partial_quadratic_bezier_points(
-                vm_points[i1:i2],
-                lower_residue,
-                upper_residue,
+                vm_points[i1:i2], lower_residue, upper_residue
             )
-            new_points[:i1] = tup[0]
-            new_points[i1:i4] = tup
-            new_points[i4:] = tup[2]
-            new_points[nppc:] = new_points[nppc - 1]
+
+            # TODO: This is not really clean but produces no gaps in circles
+            # We need a way to not have so many duplicate points without destroying interpolation!
+            new_points[:nppc] = tup
+            new_points[nppc:] = tup[1]
+            # residues = np.linspace(lower_residue,upper_residue,num_curves+1, dtype=np.float128)
```

Comment:
Is the commented block still needed?

Reply:
It's a slower but better solution to the problem because the points are not overlapping. But it also leaves gaps. So it's just for reference.